library/data/loot_table/src/main/java/org/quiltmc/qsl/loot/api/event/LootTableLoadingCallback.java

```diff
@@ -49,6 +49,15 @@ private Context(ResourceManager resourceManager, LootManager manager, Identifier
 		public static Context create(ResourceManager resourceManager, LootManager manager, Identifier id, QuiltLootTableBuilder table, LootTableSetter setter) {
 			return new Context(resourceManager, manager, id, table, setter);
 		}
+
+		/**
+		 * Replace the loot table this context is for, using the same id.
```

Comment:
```suggestion
		 * Replace the loot table this context is for, using the same identifier.
```

Reply:
Should I link to the field here?

Reply:
It's probably unnecessary, considering the field is undocumented (and doesn't need documentation, IMHO).